### PR TITLE
Set $lib property on events sent

### PR DESCRIPTION
**Pull request by macobo**
Customer request: https://posthog.slack.com/archives/C02G72Y6E9H/p1633004561010400

Not sure how to best test/deploy this, support @Twixes ?

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ async function onTrack(event, settings) {
             properties: {
                 ...event.properties,
                 distinct_id: event.userId || event.anonymousId,
+                $lib: 'segment',
```
Comment (timgl) [inline index.js:21]: This is the only place this needs to happen I think?

**Comment by timgl** [comment]
I just did the same thing ha! For segment you just need to do things in their interface which I just did.

**Comment by macobo** [comment]
Ack, gonna let you guys handle it!

**Comment by macobo** [comment]
@Twixes @timgl what's the latest on this? Was a version with this fix uploaded already or...?

**Comment by timgl** [comment]
@Twixes So turns out you have to set up a meeting with segment during which they take your integration offline for new people(!) and then you can make the necessary changes. I'll try to schedule this with them.

**Pull request closed without merging**